cssom-1/Overview.bs

```diff
@@ -1275,12 +1282,20 @@ represented by the collection.
 <pre class=idl>
 partial interface mixin DocumentOrShadowRoot {
   [SameObject] readonly attribute StyleSheetList styleSheets;
+  attribute ObservableArray&lt;CSSStyleSheet> adoptedStyleSheets;
 };
 </pre>
 
 The <dfn attribute for=DocumentOrShadowRoot>styleSheets</dfn> attribute must return a {{StyleSheetList}} collection representing
 the <a>document or shadow root CSS style sheets</a>.
 
+The <a href="https://heycam.github.io/webidl/#observable-array-attribute-set-an-indexed-value">set an indexed value</a> algorithm
```

Comment:
```suggestion
The [=observable array attribute/set an indexed value=] algorithm
```

Reply:
This one, however, doesn't seem to work. The spelling looks correct, but bikeshed doesn't give me a link. I've included both in the latest revision.

Reply:
It looks like we forgot to export it; will fix.

Reply:
https://github.com/heycam/webidl/pull/997

Reply:
Thanks! I put this back to the correct link, so it'll work once the webidl PR lands.